docs/concepts/managed-resources.md

```diff
@@ -241,6 +244,23 @@ conventions][api-versioning] for the CRDs that it deploys. In short, for
 instructions for manual migration will be provided when a new version of that
 CRD schema is released.
 
+In practice, we suggest the following guideline to provider developers:
+* Every new kind has to be introduced as `v1alpha1` with no exception.
+* Breaking changes require a schema change, i.e. `v1alpha1` needs to become
```

Comment:
I think the schema change is implied here but we are saying that version bump is required
```suggestion
* Breaking changes require a version change, i.e. `v1alpha1` needs to become
```